hack/generate.sh

```diff
@@ -18,3 +19,6 @@ ${KUBEVIRT_DIR}/tools/crd-generator/crd-generator --crd-type=vm >${KUBEVIRT_DIR}
 ${KUBEVIRT_DIR}/tools/crd-generator/crd-generator --crd-type=vmrs >${KUBEVIRT_DIR}/manifests/generated/vmrs-resource.yaml
 ${KUBEVIRT_DIR}/tools/crd-generator/crd-generator --crd-type=vmpreset >${KUBEVIRT_DIR}/manifests/generated/vmpreset-resource.yaml
 ${KUBEVIRT_DIR}/tools/crd-generator/crd-generator --crd-type=ovm >${KUBEVIRT_DIR}/manifests/generated/ovm-resource.yaml
+
+(cd ${KUBEVIRT_DIR}/tools/vms-generator/ && go build)
+${KUBEVIRT_DIR}/tools/vms-generator/vms-generator --docker-prefix=$docker_prefix --docker-tag=$docker_tag --generated-vms-dir=${KUBEVIRT_DIR}/cluster/vms-examples
```

Comment:
Two things:
* Can we call the directory just `examples`?
* can you first delete the whole content of the directory before generating the examples, so that it is definitely in sync?

Reply:
@rmohr 
- done
- done, it's generated examples after `rm -rf examples/* ; make generate`